classes/Task/TaskType.php

```diff
@@ -23,30 +24,12 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
  */
-use PHPUnit\Framework\TestCase;
-use PrestaShop\Module\AutoUpgrade\Log\Logger;
-use PrestaShop\Module\AutoUpgrade\Log\StreamedLogger;
 
-class StreamedLoggerTest extends TestCase
-{
-    /**
-     * @dataProvider filtersProvider
-     */
-    public function testFiltersProperlyApplied($level, $filterLevel, $expected)
-    {
-        $logger = new StreamedLogger();
-        $logger->setFilter($filterLevel);
-        $this->assertSame($expected, $logger->isFiltered($level));
-    }
+namespace PrestaShop\Module\AutoUpgrade\Task;
 
-    public function filtersProvider()
-    {
-        return [
-            [Logger::EMERGENCY, Logger::INFO, false],
-            [Logger::INFO, Logger::EMERGENCY, true],
-            [Logger::ERROR, Logger::ERROR, false],
-            [Logger::ERROR, Logger::WARNING, false],
-            [Logger::ERROR, Logger::CRITICAL, true],
-        ];
-    }
+class TaskType
+{
+    const TASK_TYPE_BACKUP = 'backup';
+    const TASK_TYPE_UPGRADE = 'upgrade';
```

Comment:
Is there an impact to use "update" from now?
I guess the event name will change, but this can be handled by @gericfo